helm/prometheus-rules/templates/platform/atlas/alerting-rules/mimir-to-grafana-cloud-exporter.rules.yml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: mimir-to-grafana-cloud-exporter.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: mimir-to-grafana-cloud-exporter
+    rules:
+    - alert: MimirToGrafanaCloudExporterDown
+      annotations:
+        description: '{{`Prometheus Mimir to Grafana-Cloud is down.`}}'
+        opsrecipe: mimir-grafana-cloud-exporter-failing/
+        dashboard: iWowmlSmk/prometheus?var-cluster=mimir-to-grafana-cloud
+      expr: up{job="mimir/mimir-to-grafana-cloud"} == 0
+      for: 30m
```

Comment:
30mn feels a bit short, IIUC Mimir keeps data long term, so having the mimir-to-grafana-cloud down for 30mn is not a big deal until it comes back up and sends data again.

Reply:
Let's check the prometheus data retention, because if this breaks, it will not request all data. Remote read is instant data

Reply:
`Remote read is instant data` ? 
Do you mean Prometheus is requesting mimir data in WAL only ?!

Reply:
No but the executed queries are executed now so it's requesting for instant vectors and not over Times (unless we use a rate) so there's no retries and such

Reply:
So as long as Mimir and thé prometheus are up, then we have data, it will be sent to grafana cloud eventually, but if one of the two is not working then data is gone when it's not there